test-cases/terraform/aws/encryption/in-transit/ecs_task_definition_not_encrypted_in_transit/results_summary.json

```diff
@@ -4,5 +4,5 @@
   "kics": "no",
   "snyk": "no",
   "terrascan": "no",
-  "tfsec": "no"
+  "tfsec": "yes"
```

Comment:
This also isn't a yes from what I can see - tfsec alerts about the EFS itself not being encrypted, but is not alerting about the traffic _to_ the EFS. See _transitEncryption_ here: https://docs.aws.amazon.com/AmazonECS/latest/userguide/task_definition_parameters.html

Reply:
my bad - will sort now too